src/styles/layout.scss

```diff
@@ -202,6 +202,38 @@ pre[class*='language-'] > code[class*='language-'] + h2 {
 
 /* End: Syntax Highlight Overrides */
 
+.overflow-scroll {
+  overflow-x: auto;
+}
+
+table {
+  border-collapse: collapse;
+  width: 100%;
+}
+
+thead {
+  tr {
+    td {
+      padding: var(--space-32) var(--space-14) var(--space-16);
+    }
+  }
+}
+
+tbody {
+  tr {
+    background-color: var(--black0);
+    border-bottom: 1px solid #e2e3e7;
+
+    // &:hover {
```

Comment:
remove unused code

Reply:
Thanks for the input. I am still working on this, will get it done tomorrow 👍